site/docs/v1/tech/apis/hosted-backend.adoc

```diff
@@ -0,0 +1,398 @@
+---
+layout: doc
+title: Hosted Backend APIs
+description: APIs that you to perform OAuth2 Authorization Code Flow login, logout, and refresh for applications
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.45.0
+====
+
+These APIs provide a pre-built solution for getting your app up and running using the OAuth2 Authorization Code grant with PKCE. We have in the past shown you how to link:/blog/2021/11/11/how-to-authenticate-your-react-app#create-the-express-server[create these endpoints yourself] but this solution allows you to get going with your app without writing any backend code. You just need FusionAuth!
+
+* <<Prerequisites>>
+* <<Login>>
+* <<Register>>
+* <<Callback>>
+* <<Refresh>>
+* <<Me>>
+* <<Logout>>
+
+== Prerequisites
+
+Be sure to review the link:/docs/v1/tech/core-concepts/applications#oauth[Applications] section of the FusionAuth user guide to ensure proper configuration before using the hosted endpoints.
+
+These endpoints will set the following cookies which are `Secure` and have a `SameSite` value of `Lax`. This follows our link:/learn/expert-advice/oauth/oauth-token-storage#client-side-storage[expert advice on client-side storage].
+
+.Cookies Set By the Hosted Backend
+[cols="1,1,3"]
+|===
+| Name | HttpOnly | Description
+
+| app.at
+| true
+| The access token for the configured application. This is a JWT and can be presented to your APIs to access data and functionality.
+
+| app.rt
+| true
+| The refresh token for the configured application. Only present if the `offline_access` scope is requested. This can be presented to FusionAuth to retrieve a new access token.
+
+| app.idt
+| false
+| The Id token for the user for the configured application. Only present if the `openid` scope is requested. This is a JWT and can be accessed by JavaScript to display user account information.
+
+| app.at_exp
+| false
+| The unix epoch timestamp indicating when the access token will expire. This can be checked by JavaScript to determine when a refresh token should be used to get a new access token.
+
+|===
+
+FusionAuth will set the domain on these cookies to `.example.com` where `example` is the domain name that FusionAuth is serving from either from the domain or any subdomain, `com` is the top-level domain, and the `.` allows the cookie to match the domain and all subdomains. If the host is a simple host name or IP address FusionAuth will set the domain to that (i.e. `localhost` or `127.0.0.1`).
```

Comment:
```suggestion
FusionAuth will set the domain on these cookies to `.example.com` where `example` is the domain name that FusionAuth is serving from either from the domain or any subdomain, `com` is the top-level domain, and the `.` allows the cookie to match the domain and all subdomains. If the host is a simple host name or IP address FusionAuth will set the domain to that (i.e. `localhost` or `127.0.0.1`). If FusionAuth is on a nested domain, then it will set cookies on the broadest domain that is not a top-level domain.
```